run_tests.py

```diff
@@ -300,6 +275,10 @@ def test_isort_uses_pyproject_toml_if_available(self):
             'from sys import path\n',
             isort_config=''
         )
+        # remove the .isort.cfg file
+        isortcfg_path = file_path.split('/')[: -1]
```

Comment:
I guess we should add tests for all `isort` configuration files:
- `.isort.cfg`
- `tox.ini`
- `setup.cfg`
- `pyproject.toml`

And then we can be sure that it is working properly! 👍